tox.ini

```diff
@@ -0,0 +1,46 @@
+[tox]
+isolated_build = true
+minversion: 3.14.1
+skip_missing_interpreters = true
+envlist = format-check, py{27,35,36,37,38}
+
+[tox:.package]
+# NB: tox will use the same python version as under what tox is installed to package, so unless
+# this is Python 3 you can require a given python version for the packaging environment via the
+# basepython key. We need this for Poetry, which only runs under python3 but works for python2
+# projects.
+basepython = python3
+
+[testenv]
+deps =
+    flake8
+    # NB: we can't use Pytest 5 or pytest-flakedir 2.* because these only support Python 3.
+    pytest==4.6.6
+    pytest-flake8dir==1.3.0
+commands =
+    pytest {posargs}
+    # NB: Flake8 results depend upon which Python version is used. We want it to run for every
+    # Python version to ensure that we are correctly handing the AST.
+    flake8 .
+
+[testenv:format-run]
+skip_install = true
+deps =
+    isort
+    black
+commands =
+    isort flake8_pantsbuild.py flake8_pantsbuild_test.py --apply
+    black .
+
+[testenv:format-check]
+skip_install = true
+deps =
+   {[testenv:format-run]deps}
+commands =
+    isort flake8_pantsbuild.py flake8_pantsbuild_test.py --check-only
+    black . --check
+
+[flake8]
+max_line_length: 100
+extend_ignore =
+    E203
```

Comment:
Which line triggers this? Afaict all `:` in python code above not embedded in strings or comments are not preceded by whitespace.

Reply:
Huh, not an issue anymore. Cool. I think it was necessary when using Flit and I carried it over unnecessarily.